pkg/solver/matcher/match.go

```diff
@@ -123,16 +227,17 @@ func matchOffers(
 		}
 	}
 
+	moduleID, err := data.GetModuleID(jobOffer.Module)
+	if err != nil {
+		return &moduleIDError{
+			jobOffer:      jobOffer,
+			resourceOffer: resourceOffer,
+			err:           err,
+		}
+	}
+
```

Comment:
why did you move this?

Reply:
Moved it to access it in the price mismatch case:

https://github.com/Lilypad-Tech/lilypad/blob/60f45b41b0cdbf028e21025d24b3d06148a525be/pkg/solver/matcher/match.go#L269-L273

This price mismatch case is one scope out from where we were getting the module ID. Moving it avoids recomputing the module ID for this case.